### PR TITLE
Document instanceOptions

### PR DESCRIPTION
**Pull request by ngfreiter**
Document the `instanceOptions` added to the config in https://github.com/FHIR/sushi/pull/937.

### CODE REVIEW

File: content/docs/SUSHI/configuration/_index.md [hunk 1]
```diff
@@ -141,6 +141,7 @@ The table below lists all configuration properties that can be used in SUSHI's *
 | menu | N/A | Used to generate the `fsh-generated/includes/menu.xml` file. The key is the menu item name and the value is the URL. Menus can contain sub-menus, but the IG Publisher currently only supports sub-menus one level deep. <br><br> Authors can provide their own `menu.xml` by removing this property and placing a `menu.xml` file in `/input/includes` |
 | FSHOnly | N/A | When this flag is set to `true`, no IG specific content will be generated, SUSHI will only convert FSH definitions to JSON files. When false or unset, IG content is generated.
 | applyExtensionMetadataToRoot | N/A | When set to true, the "short" and "definition" field on the root element of an Extension will be set to the "Title" and "Description" of that Extension. Default is true.
+| instanceOptions | N/A | A `key: value` pair, where key is either `setMetaProfile` (options are `always`, `never`, `inline-only`, `standalone-only`) or `setId` (options are `always` or `standalone-only`).
```
Comment (jafeltra) [inline content/docs/SUSHI/configuration/_index.md:144]: I feel like I read the "key is either" as the key can be one of these options, but not both. I tried changing it slightly to imply you can have any of the (two) keys described. Feel free to take or leave this.

```suggestion
| instanceOptions | N/A | A `key: value` pair, where key can be `setMetaProfile` (options are `always`, `never`, `inline-only`, `standalone-only`) or `setId` (options are `always` or `standalone-only`). 
```

**Review by jafeltra — COMMENTED** [review]
This looks good to me! I had one minor suggestion below.

Also, should we wait until the next SUSHI release to merge this PR?

**Review by jafeltra — APPROVED** [review]
Lookin' good!

⚠️ Don't merge until we have a new SUSHI release ⚠️

**Review by julianxcarter — APPROVED** [review]
Looks good!

**Pull request merged**